src/Quartz.Serialization.SystemTextJson/Converters/CalendarConverter.cs

```diff
@@ -88,6 +86,11 @@ public override void Write(Utf8JsonWriter writer, ICalendar value, JsonSerialize
 
             writer.WriteString(options.GetPropertyName("Description"), value.Description);
 
+            if (value is BaseCalendar baseCalendar)
```

Comment:
This change was done to generate the same JSON between STJ & Newtonsoft. See previous comment for more info.